File: DashboardDemo/DashboardSampleVB/Dashboard/DashboardControl.vb
Imports Telerik.WinControls

Public Class DashboardControl
    Inherits RadControl

    Private _element As DashboardElement

    Public ReadOnly Property Element As DashboardElement
        Get
            Return Me._element
        End Get
    End Property

    Protected Overrides Sub CreateChildItems(ByVal parent As RadElement)
        _element = New DashboardElement()
        parent.Children.Add(element)
    End Sub
End Class

File: DashboardDemo/DashboardSample/Dashboard/DashboardControl.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Telerik.WinControls;

namespace TicketTest
{
    public class DashboardControl: RadControl
    {
        DashboardElement element;

        public DashboardElement Element { get { return this.element; } }

        protected override void CreateChildItems(RadElement parent)
        {
            element = new DashboardElement();
            parent.Children.Add(element);
        }
    }
}
